pkg/morph/client/notary.go

```diff
@@ -347,6 +348,43 @@ func (c *Client) NotaryInvokeNotAlpha(contract util.Uint160, await bool, fee fix
 	return err
 }
 
+// CallWithAlphabetWitness sends transaction calling given method of specified
+// contract that requires Alphabet witness. If transaction is accepted for
+// processing, CallWithAlphabetWitness waits for it to be successfully executed.
+// Waiting is done within ctx, [ErrTxAwaitTimeout] is returned when it is done.
+func (c *Client) CallWithAlphabetWitness(ctx context.Context, contract util.Uint160, method string, args []any) error {
+	nonce := rand.Uint32()
+
+	notaryActor, mainTx, fbTx, vub, err := c.sendNotaryRequest(false, false, contract, nonce, nil, method, args...)
+	if err != nil {
+		if alreadyOnChainError(err) {
+			c.logNotaryCallAlreadyOnChain(method, nonce, vub, mainTx, fbTx)
+			return nil
+		}
+		return err
+	}
+
+	c.logNotaryCall(method, vub, mainTx, fbTx)
+
+	aer, err := notaryActor.WaitAny(ctx, vub, mainTx, mainTx)
```

Comment:
But why not `notaryActor.WaitSuccess()` that is done by `notaryInvoke()` with `await` anyway?

Reply:
Good point. A part of https://github.com/nspcc-dev/neo-go/issues/3026.